test/parallel/test-cluster-bind-twice.js

```diff
@@ -68,8 +68,9 @@ if (!id) {
 
   a.on('message', common.mustCall((m) => {
     if (typeof m === 'object') return;
-    assert.strictEqual(m, 'READY');
-    b.send('START');
+    assert(m.startsWith('READY '), 'message should start with "READY "');
+    const port = m.substr(6);
```

Comment:
Nit, I think `const port = m.substr('READY '.length)` is cleaner - but it's pretty obvious given the assert in the line above anyway.